core/src/test/scala/org/apache/spark/deploy/SparkHadoopUtilSuite.scala

```diff
@@ -77,6 +80,51 @@ class SparkHadoopUtilSuite extends SparkFunSuite with Matchers {
     })
   }
 
+  test("test expanding glob path") {
```

Comment:
IIUC, the new feature is disabled as default since `spark.sql.sources.parallelGetGlobbedPath.numThreads` is 0.

I am afraid these test causes are executed only with disabling the new feature.

Reply:
```
IIUC, the new feature is disabled as default since spark.sql.sources.parallelGetGlobbedPath.numThreads is 0.
```
Yes that's right.

```
I am afraid these test causes are executed only with disabling the new feature.
```
These mainly test the correctness of `sparkHadoopUtil.expandGlobPath`, maybe it's necessary to keep.

Reply:
I think these test should be executed with enabling and disabling the new feature.

Reply:
No problem, I'll fix this.